package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@js-temporal/polyfill",
-  "version": "0.3.0",
+  "version": "0.4.1",
```

Comment:
I don't know the exact reason why this was `v0.3.0`. But I guess it better to match` package.json`'s version.